### PR TITLE
chore(strings): Add strings needed for update password reset flow

### PR DESCRIPTION
**Pull request by vbudhram**
This PR adds strings needed for #3559

### CODE REVIEW

File: app/scripts/lib/strings.js [hunk 1]
```diff
@@ -31,6 +31,22 @@ define(function (require, exports, module) {
...
+  t('Enter new password');
+  t('Once you submit your new password, you will lose any Sync data that is not on one of your devices.');
+  t('Must be at least 8 characters');
+  t('Submit new password');
+  t('Click on the link we\'ve emailed you at %(email)s within the next hour to create a new password.');
+  t('Remember password? Sign in.');
+  t('Forgot password');
+  t('When you reset your password, you will lose any Sync data that is not on one of your devices.');
+  t('Learn how Sync works.');
+  t('Reset Password');
```
Comment (shane-tomlinson) [inline app/scripts/lib/strings.js]: Is `P` intentionally capitalized here?

**Comment by shane-tomlinson** [comment]
@vbudhram - aside from the one question, looks good. If the uppercase `P` is intentional, merge it!

File: app/scripts/lib/strings.js [hunk 2]
```diff
@@ -31,6 +31,22 @@ define(function (require, exports, module) {
   t('Account picture');
   t('%(permissionName)s (required)');
 
+  // Needed for Password flow update PR #3559
+  t('Enter new password');
+  t('Once you submit your new password, you will lose any Sync data that is not on one of your devices.');
+  t('Must be at least 8 characters');
+  t('Submit new password');
+  t('Click on the link we\'ve emailed you at %(email)s within the next hour to create a new password.');
```
Comment (vladikoff) [inline app/scripts/lib/strings.js]: I hope the "we\\`ve" won't make the whole l10n system go crazy

Comment (shane-tomlinson) [inline app/scripts/lib/strings.js]: Good point. I believe that should be extracted as just a `'`, but we should verify.

File: app/scripts/lib/strings.js (hunk 1 above)
Comment (vbudhram) [inline app/scripts/lib/strings.js]: Yea, it is for a button :)

File: app/scripts/lib/strings.js (hunk 2 above)
Comment (vbudhram) [inline app/scripts/lib/strings.js]: Do you know how I could verify it gets extracted correctly?

Comment (vladikoff) [inline app/scripts/lib/strings.js]: @vbudhram see this script: https://github.com/mozilla/fxa-content-server-l10n/blob/master/scripts/extract_strings.sh#L72 

I'll check if it works in a bit.

Comment (shane-tomlinson) [inline app/scripts/lib/strings.js]: Extracts as:

```
#: app/scripts/lib/strings.js:39
msgid ""
"Click on the link we've emailed you at %(email)s within the next hour to "
"create a new password."
msgstr ""
```

**Pull request merged**